Formula/squashfuse.rb

```diff
@@ -0,0 +1,45 @@
+class Squashfuse < Formula
+  desc "FUSE filesystem to mount squashfs archives"
+  homepage "https://github.com/vasi/squashfuse"
+  url "https://github.com/vasi/squashfuse/archive/0.1.103.tar.gz"
```

Comment:
We should be using https://github.com/vasi/squashfuse/releases/download/0.1.103/squashfuse-0.1.103.tar.gz and avoiding the Autotools dependencies.